src/sentry/static/sentry/app/views/projectDetail/projectDetail.tsx

```diff
@@ -35,11 +40,13 @@ type RouteParams = {
 
 type Props = RouteComponentProps<RouteParams, {}> & {
   organization: Organization;
+  // HoC vvv
```

Comment:
Please what does mean `vvv`?

Reply:
It's to indicate that props downwards come from HoC but I guess we don't need it 👍